admin/database/database.go

```diff
@@ -213,6 +213,12 @@ type DB interface {
 	UpsertVirtualFile(ctx context.Context, opts *InsertVirtualFileOptions) error
 	UpdateVirtualFileDeleted(ctx context.Context, projectID, branch, path string) error
 	DeleteExpiredVirtualFiles(ctx context.Context, retention time.Duration) error
+
+	FindBillingOrganizationIDs(ctx context.Context) ([]string, error)
```

Comment:
I first thought this returned billing customer IDs. Can we call with `FindOrganizationIDsWithBilling` or something like that?